device/mellanox/x86_64-mlnx_msn2700-r0/plugins/eeprom.py

```diff
@@ -45,12 +54,11 @@ def __init__(self, name, path, cpld_root, ro):
                 time.sleep(1)
             else:
                 break  
-                      
         if not (os.path.exists(EEPROM_SYMLINK) or os.path.isfile(CACHE_FILE)):
             log_error("Nowhere to read syseeprom from! No symlink or cache file found")
             raise RuntimeError("No syseeprom symlink or cache file found")
 
-        self.eeprom_path = EEPROM_SYMLINK
+        self.eeprom_path = EEPROM_SYMLINK if 'simx' in onie_platform else CACHE_FILE
```

Comment:
@mykolaf Looks like we a bug here:
If platform is SimX we will use regular EEPROM_SYMLINK 
If platform is not SimX we will use CACHE_FILE